packages/http-client-csharp/emitter/src/emitter.ts

```diff
@@ -62,6 +65,13 @@ export async function $onEmit(context: EmitContext<CSharpEmitterOptions>) {
   const logger = new Logger(program, options.logLevel ?? LoggerLevel.INFO);
 
   if (!program.compilerOptions.noEmit && !program.hasError()) {
+    const defaultSDKContextOptions: CreateSdkContextOptions = {
```

Comment:
nit: just inline this in the one place it is used.

Reply:
remove the default option since TCGC has the same default values
https://github.com/Azure/typespec-azure/blob/dc0aafe33992e17857f757aeed0290cac70c459c/packages/typespec-client-generator-core/src/context.ts#L130
https://github.com/Azure/typespec-azure/blob/dc0aafe33992e17857f757aeed0290cac70c459c/packages/typespec-client-generator-core/src/context.ts#L48